test/llvm/lifetime-intrinsics/lifetime-proc.chpl

```diff
@@ -0,0 +1,34 @@
+// This test is used to verify that llvm.lifetime instrinsics are being emitted
+// for stack variables of basic type inside a procedure
+
+// CHECK: %[[REG1:[0-9]+]] = bitcast i64* %x_chpl to i8*
```

Comment:
I'd recommend moving these CHECK comments inside of the functions they apply to.